lib/gstreamer/msdk/transcoder.py

```diff
@@ -56,6 +56,9 @@ class TranscoderTest(BaseTranscoderTest):
       Codec.VP9 : dict(
         hw = (platform.get_caps("decode", "vp9_8"), have_gst_element(f"msdkvp9dec"), f"matroskademux ! vp9parse !  msdkvp9dec"),
       ),
+      Codec.AV1 : dict(
+        hw = (platform.get_caps("decode", "av1_8"), have_gst_element(f"msdkav1dec"), f"ivfparse ! av1parse ! msdkav1dec"),
```

Comment:
Should use "matroskademux" instead of "ivfparse".  That is, it should be the inverse operation as the encode pipeline.